helpers/deploy/deploymentConfig.ts

```diff
@@ -160,6 +160,18 @@ export const config: Config = {
       { method: "setMaxDailyReceiveLimit(uint256)", args: [100] },
     ],
   },
+  opmainnet: {
+    methods: [
+      { method: "setMinDstGas(uint16,uint16,uint256)", args: [LZ_CHAINID["opmainnet"], 0, 200000] },
```

Comment:
The argument should be the LZ id of the destination network. Given we have a mesh architecture, I think we should an entry in the array for each supported network:

```suggestion
      { method: "setMinDstGas(uint16,uint16,uint256)", args: [LZ_CHAINID["ethereum"], 0, 200000] },
      { method: "setMinDstGas(uint16,uint16,uint256)", args: [LZ_CHAINID["bscmainnet"], 0, 200000] },
      { method: "setMinDstGas(uint16,uint16,uint256)", args: [LZ_CHAINID["opbnbmainnet"], 0, 200000] },
      { method: "setMinDstGas(uint16,uint16,uint256)", args: [LZ_CHAINID["arbitrumone"], 0, 200000] },
```

@GitGuru7

Reply:
How about adding zksync as well?

Reply:
I agree. @Debugger022, could you please add it?

Reply:
I would also update the entries in the rest of the networks, adding the missing items, to reflect the mesh architecture we have

Reply:
@GitGuru7 , I think we should add the support only for bsc network

Reply:
@Debugger022 Absoulutely, @chechu, Given this is the config for multichain governance, I  think we don't need mesh architecture here.

Reply:
You are right, my fault